kuzzleio/kuzzle-admin-console
# Dynamic list width

## What does this PR do ?
This PR tweaks the width of the list views (only in the Data section, for the moment) in order to make them narrower on large screens (which seemed not very meaningful and ugly).
It also introduces some horizontal layout fixes to bugs that showed up in my 13' Mac screen.

### How should this be manually tested?

* Open a collection on a large screen
* You should see that the document list (and the Watch section) are narrower than the window and have some whitespace on the right

### Other changes
* Changing the width of the list containers also showed some layout weakness in some components. Some `col s* m* l* xl*` have been adjusted.

### Boyscout

* Added the missing `vue2-leaflet` package
* Added some `:key` attributes to elements that had `v-for`
